runtime/westend/src/lib.rs

```diff
@@ -319,6 +319,39 @@ impl pallet_session::historical::Config for Runtime {
 	type FullIdentificationOf = pallet_staking::ExposureOf<Runtime>;
 }
 
+parameter_types! {
+	// no signed phase for now, just unsigned.
+	pub const SignedPhase: u32 = 0;
+	pub const UnsignedPhase: u32 = EPOCH_DURATION_IN_BLOCKS / 4;
```

Comment:
FYI this is same as ElectionLookAhead

Reply:
I treat westend as a testnet :P but will make it same as polkadot now.